src/runtime/config/configuration-clh.toml.in

```diff
@@ -40,7 +40,7 @@ rootfs_type=@DEFROOTFSTYPE@
 # * Intel TDX
 #
 # Default false
-# confidential_guest = true
+confidential_guest = true
```

Comment:
Will need to make this a variable for vanilla.

Reply:
In this case I am tempted to create a new *.in file, instead of modifying configuration-clh.toml.in.

That's to be consistent with the upstream convention - they have these separate files:

configuration-qemu.toml.in
configuration-qemu-tdx.toml.in

It seems like our file should be named:

configuration-clh-snp.toml.in

Reply:
Yep we do have a configuration-clh-snp.toml.in file in this change.

Reply:
Out of those two, I vote to create the third file dynamically in the spec since it's a trivial operation.

But also, since this is only used for debugging, I'm thinking we shouldn't package that third config at all. It would add another step to the debugging process, but this could be done as part of a broader debug script and could be an opportunity to streamline our dev process. Food for thought.